apigee_edge.install

```diff
@@ -389,3 +389,11 @@ function apigee_edge_update_9002() {
   $edge_settings['content']['callbackUrl'] = $new_edge_settings['content']['callbackUrl'];
   $config_storage->write('core.entity_view_display.developer_app.developer_app.default', $edge_settings);
 }
+
+/**
+ * Install Field UI module to maintain dependency.
+ */
+function apigee_edge_update_9003() {
```

Comment:
The update hook for Drupal 10+ should start with 10.

https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_N/10

(Probably old update hooks should have been removed when 3.x branch started and Drupal core major version was bumped)

Reply:
> The update hook for Drupal 10+ should start with 10.

Thanks! Noted 👍 

> (Probably old update hooks should have been removed when 3.x branch started and Drupal core major version was bumped)

We will take care regarding this next time.